packages/canary-web-components/src/components/ic-table-of-contents/ic-table-of-contents.css

```diff
@@ -0,0 +1,135 @@
+/*   --ic-table-of-contents-text: var(--ic-color-text-primary);
+  --ic-table-of-contents-label: var(--ic-color-text-primary);
+  --ic-table-of-contents-banner: var(--ic-action-default);
+  --ic-table-of-contents-banner-monochrome: var(--ic-action-monochrome-dark);
+  --ic-table-of-contents-hover: var(--ic-state-layers-action-bg-hover);
+  --ic-table-of-contents-hover-monochrome: var(--ic-state-layers-action-bg-dark-hover);
+  --ic-table-of-contents-pressed: var(--ic-state-layers-action-bg-pressed);
+  --ic-table-of-contents-pressed-monochrome: var(--ic-state-layers-action-bg-dark-pressed);
+   */
+
+:host {
+  display: block;
+  border: none;
+  max-height: 100vh;
```

Comment:
<img width="665" height="197" alt="Screenshot 2025-08-08 at 12 31 09" src="https://github.com/user-attachments/assets/27a63879-5440-4351-898c-faf777ba4fb4" />

Could we add a z-index to table of contents? I think that would fix the tooltip going behind the content.